src/components/RightSidebar/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -20,59 +20,75 @@
 -->
 
 <template>
-	<div>
+	<div class="matterbridge-settings">
 		<div v-if="loading" class="loading" />
 		<div v-show="!loading">
+			<h3>
+				<span class="icon icon-category-integration" />
+				<p>{{ t('spreed', 'Bridge with other services') }}</p>
+			</h3>
 			<div id="matterbridge-header">
-				<h3>
-					{{ t('spreed', 'Bridge with other services') }}
-				</h3>
 				<p>
 					{{ t('spreed', 'You can bridge channels from various instant messaging systems with Matterbridge.') }}
 					<a href="https://github.com/42wim/matterbridge/wiki" target="_blank" rel="noopener">
-						{{ t('spreed', 'More info on Matterbridge.') }}
+						<span class="icon icon-external" />
+						{{ t('spreed', 'More info on Matterbridge') }}
 					</a>
 				</p>
 			</div>
 			<div class="basic-settings">
-				<ActionCheckbox
-					:token="token"
-					:checked="enabled"
-					@update:checked="onEnabled">
-					{{ t('spreed', 'Enabled') }}
-					({{ processStateText }})
-				</ActionCheckbox>
-				<button class="" @click="showLogContent">
-					{{ t('spreed', 'Show matterbridge log') }}
-				</button>
-				<Modal v-if="logModal"
-					@close="closeLogModal">
-					<div class="modal__content">
-						<textarea v-model="processLog" class="log-content" />
-					</div>
-				</Modal>
-				<Multiselect
-					ref="partMultiselect"
-					v-model="selectedType"
-					label="displayName"
-					track-by="type"
-					:placeholder="newPartPlaceholder"
-					:options="formatedTypes"
-					:internal-search="true"
-					@input="clickAddPart" />
+				<div class="add-part-wrapper">
+					<span class="icon icon-add" />
+					<Multiselect
+						ref="partMultiselect"
+						v-model="selectedType"
+						label="displayName"
+						track-by="type"
+						:placeholder="newPartPlaceholder"
+						:options="formatedTypes"
+						:user-select="true"
+						:internal-search="true"
+						@input="clickAddPart">
+						<template #option="{option}">
+							<span :class="option.icon" />
+							{{ option.displayName }}
+						</template>
+					</Multiselect>
+				</div>
 				<ActionButton
```

Comment:
can you make this a button, disabled when unchanged, only getting enabled when content was changed, so it's much clearer you need to save things?

Reply:
We can also show it only after changes (add, edit, delete a part). I pushed it like that.

ActionButton has a disabled state but it's visually not very clear it's disabled. Making this a real button would not look as nice imho.

Do you still prefer a button with disabled text?

Reply:
Well ActionButton should only be used in an Actions-menu. So just use a plain `<button>` html element here.

That button can appear once you changed something. (Although I'm currently not sure. Since this will all be moved to the new settings dialog anyway, right?)

Reply:
Right, no more ActionButton (and ActionCheckbox) there.

> Since this will all be moved to the new settings dialog anyway, right?

Why would it change the save button apparition conditions?